src/screens/lib/navigationService.ts

```diff
@@ -0,0 +1,31 @@
+import { createSecureStorage } from '@shared/lib';
+
+import { InitialRoute } from './types';
+
+const storage = createSecureStorage('navigation-storage');
+
+function SessionService() {
```

Comment:
Shouldn't it be named the same as file name?

Reply:
Correct